src/connection/sim.ts

```diff
@@ -475,6 +494,21 @@ export class SimulatorPlugin implements Connection {
       initial = undefined;
       updateRate = 100;
       cls = RampPv;
+    } else if (nameInfo.protocol.includes("sim://flipflop")) {
+      // Default rate if not configured with flipflop(X)
+      updateRate = 1000;
+      const strCmd = nameInfo.protocol;
+      if (strCmd.includes("(") && strCmd.includes(")")) {
+        const intevalSecStr = strCmd.slice(
+          strCmd.indexOf("(") + 1,
+          strCmd.lastIndexOf(")")
+        );
```

Comment:
I feel a regular expression match here could be more robust. Have you considered this? There could be reasons not to prefer it e.g. speed but I think this is only called on creation.

Reply:
Yes this is only called once on the creation of the simulated PV so I think regex would be more robust. I have made this change in the latest commit.